src/HtmlServiceProvider.php

```diff
@@ -33,16 +33,17 @@ public function register()
      *
      * @return void
      */
-    protected function registerIlluminateSession(){
-        if (!isset($this->app['session.store'])) {
+    protected function registerIlluminateSession()
+    {
+        if (!isset($this->app->availableBindings['session.store'])) {
             $this->app->configure('session');
             
-            if(!$this->app['config']->has('session.driver')) {
+            if (!$this->app['config']->has('session.driver')) {
                 $this->app['config']->set('session.driver', 'file');
-            } 
+            }
 
-            $this->app->register(\Illuminate\Session\SessionServiceProvider::class);
-        }         
+            $this->app->register('Illuminate\Session\SessionServiceProvider');
```

Comment:
Here we used `'Illuminate\Session\SessionServiceProvider'` instead of `\Illuminate\Session\SessionServiceProvider::class` to allow support for PHP 5.4.0